config/hsc/processCcd.py

```diff
@@ -19,6 +19,7 @@
 config.calibrate.astrometry.matcher.maxRotationDeg=1.145916
 config.calibrate.astrometry.wcsFitter.numRejIter=3
 
+config.charImage.astrometry.wcsFitter.order = 3
 config.calibrate.astrometry.wcsFitter.order = 3
```

Comment:
Is it always true that these should be the same?  I presumed there may conceivably be some use case for allowing more/less freedom in the characterize vs. calibrated stages.  I'm happy to make the implied desire for using the same value for both here.